pages/calculator/[slug].js

```diff
@@ -54,8 +54,20 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }) {
 
   const isFirstPage = () => page.slug === 'head-initial-1-cont';
 
+  const url = "https://clearviction.org/"
+
   const theme = useTheme();
-  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  const matchesXS = useMediaQuery(theme.breakpoints.down('sm'));
+
+  function closeDialog() {
```

Comment:
Can you update this function to be an arrow function like the `handleClose` func below for consistency. Also in `ShareButtons`, please update the `handleEmailClick func`, since the func right above it is an arrow function.

Reply:
Got you!